proto/process/agent.proto

```diff
@@ -85,6 +87,10 @@ message CollectorConnections {
 	string platform = 19;
 	string platformVersion = 20;
 
+	// host configuration, ephemeral port range
+	int32 ephemeralPortRangeStart = 32;
+	int32 ephemeralPortRangeEnd = 33;
+
```

Comment:
@hmahmood aren't you using these identifiers in https://github.com/DataDog/agent-payload/pull/99/files

Reply:
Yes, I can change them once this merges.